api/resolver/query/project.go

```diff
@@ -0,0 +1,25 @@
+package query
+
+import (
+	"context"
+	"github.com/robojones/graphql/lib/session_context"
+	"github.com/robojones/graphql/prisma"
+)
+
+func (resolver *QueryResolver) Projects(ctx context.Context) ([]prisma.Project, error) {
+	user, err := session_context.User(ctx)
+
+	if err != nil {
+		return make([]prisma.Project, 0), err
+	}
+
+	projects, err := resolver.Prisma.User(prisma.UserWhereUniqueInput{
+		ID: &user.ID,
+	}).Projects(nil).Exec(ctx)
+
+	if err != nil {
```

Comment:
In theory, `return projects, nil` or even `return resolver.Prisma.User...` as well.

Reply:
This will also allow you to use one-liners in some resolvers